File: core/core-api/src/main/java/com/ssafy/witch/controller/appointment/response/AppointmentResponse.java
package com.ssafy.witch.controller.appointment.response;

import com.ssafy.witch.appointment.AppointmentStatus;
import java.time.LocalDateTime;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class AppointmentResponse {

  private String appointmentId;
  private String name;
  private LocalDateTime appointmentTime;
  private AppointmentStatus status;
  private boolean isMyAppointment;

}

File: Witch/app/src/main/java/com/ssafy/witch/data/model/response/AppointmentResponse.kt
package com.ssafy.witch.data.model.response

import com.ssafy.witch.data.model.dto.MyAppointment

data class AppointmentResponse(
    var appointmentId: String,
    var name: String,
    var status: String,
    var appointmentTime: String,
    var group: MyAppointment.Group
)
